### PR TITLE
Adding my kennard-stone package

### PR DESCRIPTION
**Pull request by yu9824**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/kennard-stone/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{% set name = "kennard-stone" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/7a/ce/a4e3b09323ed9e7ae0054795927ba62d1d9bfc5d3e09f0d92b8706a50e06/kennard_stone-1.1.1.tar.gz"
```
Comment (chrisburr) [inline recipes/kennard-stone/meta.yaml:9]: This should use the predictable form of the URL templated with `{{ version }}`. See `recipes/example/meta.yaml` for an example.

File: recipes/kennard-stone/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - numpy
+    - pandas
+    - pip
+    - python >=3.6
+    - scikit-learn
```
Comment (chrisburr) [inline recipes/kennard-stone/meta.yaml:23]: Nominally pure Python packages should only need `python` and `pip` in `host`:
```suggestion
    - pip
    - python >=3.6
```

File: recipes/kennard-stone/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - kennard_stone
+
+about:
+  home: "https://github.com/yu9824/kennard_stone"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE 
+  summary: "A method for selecting samples by spreading the training data evenly."
+  doc_url: 
+  dev_url:
```
Comment (chrisburr) [inline recipes/kennard-stone/meta.yaml:41]:
```suggestion
```

**Comment by yu9824** [comment]
Thank you for your review and comments.

I have modified the `meta.yaml` based on your point. Specifically, here is what we have done
- Changed the `url` to one that uses variables.
- Removed `doc_url:` and `dev_url:` entries.
- Changed `host:` to only pip and python.

**Pull request merged**